dashboard/src/reducers/apps.ts

```diff
@@ -30,6 +30,14 @@ const appsReducer = (
       return { ...state, isFetching: true, listingAll: action.payload };
     case getType(actions.apps.receiveAppList):
       return { ...state, isFetching: false, listOverview: action.payload };
+    case getType(actions.apps.updateAppListItem):
+      if (state.listOverview) {
```

Comment:
for also updating `selected`, we have the option here to check if `selected.releaseName === payload.releaseName` or we can pass down a parameter in the payload to know if this is coming from a `getAppWithUpdateInfo` or a `fetchAppsWithUpdateInfo`.